Source/Charts/Data/Implementations/Standard/BarChartDataSet.swift

```diff
@@ -50,68 +50,39 @@ open class BarChartDataSet: BarLineScatterCandleBubbleChartDataSet, BarChartData
     {
         _entryCountStacks = 0
         
-        for i in 0 ..< entries.count
-        {
-            if let vals = entries[i].yValues
-            {
-                _entryCountStacks += vals.count
-            }
-            else
-            {
-                _entryCountStacks += 1
-            }
-        }
+        entries.forEach { _entryCountStacks += $0.yValues?.count ?? 1 }
     }
     
     /// calculates the maximum stacksize that occurs in the Entries array of this DataSet
     private func calcStackSize(entries: [BarChartDataEntry])
     {
-        for i in 0 ..< entries.count
+        for e in entries
         {
-            if let vals = entries[i].yValues
-            {
-                if vals.count > _stackSize
-                {
-                    _stackSize = vals.count
-                }
+            guard let vals = e.yValues, vals.count > _stackSize else {
```

Comment:
not sure the code style here. put all guard sentence one line or use another line for `{`

Reply:
Ugh, yea I'll fix it.

Still find this to be an incredibly weird style for Swift. It's clear even the library was written without any consideration for Swift, and instead other language(s). I think now that there are common accepted practices, we should revisit some of ours.